src/Video-detail/video-detail.css

```diff
@@ -0,0 +1,129 @@
+.video-container{
+    width: 100%;
+    height: 390px;
```

Comment:
Avoid using px, instead rem is suggested to be used.

Reply:
Done, updated all px values to rem